src/workspacer.FSharpConfig/FSharpConfigPlugin.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace workspacer.FSharpConfig
+{
+    /// <summary>
+    /// An empty plugin that allows the F# configuration integration in the same way as other plugins.
+    /// </summary>
+    public class FSharpConfigPlugin : IPlugin
+    {
+        /// <summary>
+        /// Does nothing for the moment. In the future we may add an extra menu option to create a template.
+        /// </summary>
+        /// <param name="context">The configured configuration context.</param>
+        public void AfterConfig(IConfigContext context)
+        {
+        }
+    }
+}
```

Comment:
For now this class does nothing, and provides nothing.

If we need something like this in the future, can't we add it then, instead of adding some empty boilerplate now? :smile:

Reply:
Sadly it's mandatory to have such a class in the current plugin loader 😅